llmstudio/engine/providers/anthropic.py

```diff
@@ -76,16 +104,6 @@ async def parse_response(
                         created=int(time.time()),
                         model=kwargs.get("request").model,
                         object="chat.completion.chunk",
-                    ).model_dump(),
-                elif isinstance(
-                    chunk, anthropic.types.message_stop_event.MessageStopEvent
-                ):
-                    yield ChatCompletionChunk(
-                        id=str(uuid.uuid4()),
-                        choices=[
-                            Choice(delta=ChoiceDelta(), finish_reason="stop", index=0)
-                        ],
-                        created=int(time.time()),
-                        model=kwargs.get("request").model,
-                        object="chat.completion.chunk",
-                    ).model_dump(),
+                    ).model_dump()
+                except:
```

Comment:
this is dangerous, which errors should we explicitely allow to pass, and which ones should we raise?